### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by maebeale**

### CODE REVIEW

**Review by zph — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
   * events calendar
   * donor system
   * volunteer system
   * photo sharing / tagging /commenting
   * chat
-  * newsletter sign-up
-  * HH Website
-* Admin dashboard is provided to provide user data (number of invites sent,
+  * facebook feed
+  * instagram feed
```
Comment (mirandawang) [inline README.md:22]: I think we are including Twitter as well! :)

File: README.md [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
 
-We are building a one-stop social networking platform for members and their
-families.
+"Orchard" is a one-stop social networking platform for alumni counselors, staff, campers, and volunteers to engage with camp -- and each other!
 
 ## Setup
 ==
 - Install Ruby 2.4.1 (chruby, rbenv, rvm, etc)
-- `bundle install`
+- `bin/setup` initially, then `bin/update thereafter`
```
Comment (mirandawang) [inline README.md:8]: I think the ` is in the wrong spot here

**Review by mirandawang — COMMENTED** [review]
Looks great!! :100: 👍  I commented on two small things :)

**Pull request merged**